scrapysharp-dt2020.csproj

```diff
@@ -19,4 +19,52 @@
     <PackageReference Include="ScrapySharp" Version="3.0.0" />
   </ItemGroup>
 
+  <ItemGroup>
```

Comment:
What is the purpose of adding this item group of file names to the csproj files?  I'm not opposed to them being there, but I don't understand why to add this.

Reply:
This adds the csv files to the debug directory when I try to run the project. For me the project wouldn't run as it couldn't find those files as they weren't being copied. This could just be a thing on my machine though, happy for it to be left out!

Reply:
No, that seems reasonable.  If it breaks something, we can fix that later.